packages/usa-table/src/usa-table.stories.js

```diff
@@ -20,4 +39,19 @@ Borderless.args = BorderlessContent;
 export const Striped = Template.bind({});
 Striped.args = StripedContent;
 
+export const StickyHeader = Template.bind({});
+StickyHeader.args = {
+  ...DefaultContent,
+  ...StickyHeaderContent,
+  stickyHeader: true,
```

Comment:
Just curious, any reason why you chose to add `stickyHeader: true` here instead of in the `StickyHeaderContent` json? My initial thought was that would be more consistent with existing variant/content patterns.

Reply:
Good catch! I moved it to JSON in [Move 'stickyheader:true' to JSON](https://github.com/uswds/uswds/pull/5420/commits/0d8a54dbfe4b4610a8bcd0d89a0b5fb5746ab939).